### ISSUE TITLE
[Bug] Google Workspace accounts which require setting up a lock screen password do not work

### ISSUE THREAD

**Issue by VesperLlama** [microg/GmsCore#1726]
**Describe the bug**
My University provided Google account requires a lock screen password to be set to use it. It doesn't have any problem when I log in with MicroG but I cannot use the account on any app even after I set up a lock screen password. Apps like FairEmail or even GMail cannot access the account.

**To Reproduce**
1. Add a account with Mobile Management turned on.
2. Add a lock screen password.
3. Try to log in to that account on other apps like FairEmail, GMail or Youtube.
4. The apps show the added account but cannot access it.

**Expected behavior**
The apps should be able to access it when lock screen password is enabled.

**System**
Android Version: 12
Custom ROM: CrDroid 8.5

**Comment by aximut** [comment]
Same here, any updates or workarounds?

**Comment by Umang96** [comment]
Any update on this? Maybe something is missing in the AuthRequest?

```
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: java.io.IOException: Error=DeviceManagementScreenlockRequired
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at org.microg.gms.common.HttpFormClient.request(HttpFormClient.java:95)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at org.microg.gms.auth.AuthRequest.getResponse(AuthRequest.java:245)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at org.microg.gms.auth.AuthManager.requestAuth(AuthManager.java:282)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at org.microg.gms.auth.loginservice.AccountAuthenticator.getAuthToken(AccountAuthenticator.java:97)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at android.accounts.AbstractAccountAuthenticator$Transport.getAuthToken(AbstractAccountAuthenticator.java:244)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at android.accounts.IAccountAuthenticator$Stub.onTransact(IAccountAuthenticator.java:282)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at android.os.Binder.execTransactInternal(Binder.java:1159)
09-26 18:23:08.846  1897  4075 W GmsAuthenticator: 	at android.os.Binder.execTransact(Binder.java:1123)
```

**Comment by fynngodau** [comment]
The feature to support Workspace accounts with basic integrity is included in v0.3.4. I suggest to close this issue.

**Comment by ale5000-git** [comment]
I'm closing it, if someone has still problems with the latest version please write here.
